sources/assembly-data/ATTR_assembly.types.yaml

```diff
@@ -89,11 +109,19 @@ attributes:
         description: "LOEWE Centre for Translational Biodiversity Genomics (click to view on ENA)"
       prjna758215:
         description: "MetaInvert (click to view on ENA)"
+      PRJNA1046164:
+        description: "Ocean Genomes (click to view on ENA)"
+      PRJEB71705:
+        description: "Project Psyche (click to view on ENA)"
       prjna707598:
         description: "Squalomix (click to view on ENA)"
       prjna489243:
         description: "Vertebrate Genomes Project (click to view on ENA)"
+      PRJNA955268:
+        description: "Yggdrasil (click to view on ENA)"
   biosample:
+    description: BioSample Identifier
+    long_description: A unique identifier assigned to each entry in the NCBI BioSample database. A BioSample contains descriptions of biological source materials used in experimental assays, such as nucleotide sequencing
```

Comment:
the definition is correct, but not sure that the second sentence will help the LLM

Reply:
Removed second sentence to something museful to EBP projects